### PR TITLE
Added documentation

### PR DESCRIPTION
**Pull request by yoannpicquenot**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,44 @@
...
+# Prerequisites
+
+Program | Version
+------- | -----------
+node    | >= 6.10x
+npm     | >= 4.5x
+
+# Clone the repository
+
+```shell=
```
Comment (Quentin01) [inline README.md]: In markdown it's only `shell` and not `shell=`

File: README.md [hunk 2]
```diff
@@ -1 +1,44 @@
...
+git clone git@github.com:Lugdunum3D/LugBench-API.git
```
Comment (Quentin01) [inline README.md]: Use the https link for users, it's better.

File: README.md [hunk 3]
```diff
@@ -1 +1,44 @@
...
+```
+
+# Installation
+
+This command line below will install all dependencies you need to run the project.
+```shell=
```
Comment (Quentin01) [inline README.md]: Same remark `shell`

File: README.md [hunk 4]
```diff
@@ -1 +1,44 @@
...
+npm install
+```
+
+# Running the API in *http://localhost:5000*
+
+```shell=
+npm start
+```
+
+:::info
```
Comment (Quentin01) [inline README.md]: What are you doing with `:::info`? Do you mean `##info`?

File: README.md [hunk 5]
```diff
@@ -1 +1,44 @@
...
+The default port is 5000. If you want to set a different one for the API to listen, set a environment variable system like this:
+
+```shell=
```
Comment (Quentin01) [inline README.md]: Same remark, `shell`

File: README.md [hunk 6]
```diff
@@ -1 +1,44 @@
...
+export PORT=3000
+```
+:::
```
Comment (Quentin01) [inline README.md]: What are you doing with `:::`?

File: README.md [hunk 7]
```diff
@@ -1 +1,44 @@
...
+
+
+# Routes
```
Comment (Quentin01) [inline README.md]: It's great to have all the routes, but can't we provide also the content of the JSON? With a JSON Schema for example.

**Review by Quentin01 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (yoannpicquenot) [reply README.md]: My bad, it's Hackmd autocompletion

File: README.md (hunk 4 above)
Comment (yoannpicquenot) [reply README.md]: I thought it was right. Because on hackmd it works

File: README.md (hunk 7 above)
Comment (mapy69003) [inline README.md]: :+1:

File: README.md [hunk 8]
```diff
@@ -9,36 +16,42 @@ npm     | >= 4.5x
 
 # Clone the repository
 
-```shell=
-git clone git@github.com:Lugdunum3D/LugBench-API.git
+Download the project [here](https://github.com/Lugdunum3D/LugBench-API/archive/dev.zip)
```
Comment (saveman71) [inline README.md]: I honestly don't think any of the users might want to do that.

File: README.md [hunk 9]
```diff
@@ -9,36 +16,42 @@ npm     | >= 4.5x
...
-```shell=
+To start the API, type de command line below.
+
+```
 npm start
 ```
 
-:::info
-The default port is 5000. If you want to set a different one for the API to listen, set a environment variable system like this:
+**NOTE**: The default port is 5000. If you want to set a different one for the API to listen, set a environment variable system like this:
```
Comment (saveman71) [inline README.md]: I think NOTE is better as "Note" as it's done on our docs.

**Review by saveman71 — CHANGES_REQUESTED** [review]

File: README.md (hunk 8 above)
Comment (Quentin01) [reply README.md]: I was talking about cloning by https not downloading the project.
`git clone https://github.com/Lugdunum3D/LugBench-API.git`

**Pull request closed without merging**